pibooth/plugins/lights_plugin.py

```diff
@@ -2,83 +2,100 @@
 
 import time
 import pibooth
+from pibooth.utils import LOGGER
+from gpiozero import Device, LEDBoard
+try:
+    from RPi import GPIO
+    LOGGER.info("Start on Raspberry pi")
+except ImportError:
+    # GPIO library only existes on Raspberry Pi
+    # Set the default pin factory to a mock factory
+    from gpiozero.pins.mock import MockFactory
+    Device.pin_factory = MockFactory()
+    LOGGER.info("Programme not start on Raspberry pi use MockFactory() mode")
```

Comment:
This whole block is already done by pibooth/controls/__init__.py , why is it duplicated?